### DISCUSSION TITLE
Particles created from two different PBD particle systems do not collide with each other

### DISCUSSION THREAD

**Discussion by GJRockLin** [NVIDIA-Omniverse/PhysX#344] [Q&A] [answered]
I create two PBD particle system with each one set different restOffset and contactOffset. But the problem was that these two particle did not collide with each other. Is there any way to enable the collision of two particle systems?

**Comment by vreutskyy** [comment, marked as answer]
Hi @GJRockLin 
PBD particles can only interact with the particles in the same particle system. It implies the limitation that all interacting particles should have the same size - which is important for the performance. It seems we'll need to make it clear in the docs. Currently I couldn't find it mentioned anywhere, but that's how it has been since Flex times.
Thanks
